mauth_client/fastapi_authenticator/fastapi_authenticator.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+from fastapi import FastAPI, HTTPException, Request
+from mauth_client.authenticator import LocalAuthenticator, RemoteAuthenticator
+from mauth_client.config import Config
+from mauth_client.signable import RequestSignable
+from mauth_client.signed import Signed
+from typing import Tuple, Union
+
+logger = logging.getLogger("fastapi_mauth")
+MAuthAuthenticator = LocalAuthenticator if Config.MAUTH_MODE == "local" else RemoteAuthenticator
```

Comment:
thoughts on removing support for remote authenticator?  we'd like to deprecate the api route (authentication tickets) for calling mauth service for verification.

Reply:
I'm fine with removing it. I don't think anyone is using it

Reply:
@ykitamura-mdsol think i will leave that for another PR, this one is already big from juggling all the dependencies for python 3.10

Reply:
fair enough, thanks for adding python 3.10! we can make a cleanup PR later. 

(since we are not publishing docs to readthedocs, I'm thinking of removing https://github.com/mdsol/mauth-client-python/tree/develop/docs also.)